tests/conftest.py

```diff
@@ -176,3 +177,38 @@ def test_work_id(seed_test_data):
 @pytest.fixture(scope='session')
 def test_link_id(seed_test_data):
     return seed_test_data['link_id']
+
+@pytest.fixture(scope='session')
+def seed_unclassified_record(db_manager):
+    if db_manager is None:
+        return {'record_uuid': '7bd4a8c0-9b3a-487b-9c7c-1dd6890a5c7a'}
+
+    test_record_data = {
+        'title': 'Moby-Dick',
+        'uuid': uuid4(),
+        'frbr_status': 'to_do',
+        'cluster_status': False,
+        "source": TEST_SOURCE,
+        'authors': ['Herman Melville||true'],
+        'identifiers': ['9780142437247|isbn'],
+        'source_id': 'moby-dick-123|test',
+        'date_modified': datetime.now(timezone.utc).replace(tzinfo=None)
+    }
```

Comment:
Let's simplify! I think we can move this up to a list in the other seed fixture.